doc/examples.rst

```diff
@@ -0,0 +1,15 @@
+Examples
+========
+
+.. toctree::
+   :hidden:
+
+   EEG Example <examples/EEG_Example_output>
+
+Examples of how to use the :code:`mltsp` library to perform machine learning
+for time series data.
+
+EEG Example
```

Comment:
I thought it'd be nice for once we have multiple examples; I also like having a separate section in the table of contents, rather than having each example at the top level. Not super attached to it but I'd say leave it like this.